qcodes/instrument_drivers/Keysight/KtM960x.py

```diff
@@ -5,16 +5,26 @@
 from typing import (Dict, Optional)
 
 from qcodes import Instrument, validators as vals
+from qcodes.utils.helpers import create_on_off_val_mapping
 
 
 class KtM960x(Instrument):
+    """
+    Provide a wrapper for the Keysight KtM960x DAC. This driver provides
+    an interface into the IVI-C driver provided by Keysight.
+    The .dll is installed by default into
+        C:\Program Files\IVI Foundation\IVI\Bin\KtM960x_64.dll
+    but a different path can be supplied to the constructor
+    """
+
     _default_buf_size = 256
     _dll_loc = r"C:\Program Files\IVI Foundation\IVI\Bin\KtM960x_64.dll"
 
     def __init__(self,
                  name: str,
                  address: str,
                  options: bytes = b"",
+                 dll_string: str = _dll_loc,
```

Comment:
``
```suggestion
                 dll_path: str = r"C:\Program Files\IVI Foundation\IVI\Bin\KtM960x_64.dll"
```

This does not look correct. As far as I can see `dll_string` is never used.
I would suggest setting the argument equal to a default value of r"C:\Program Files\IVI Foundation\IVI\Bin\KtM960x_64.dll" I would also call it dll_path.

Reply:
and then update line 26 to use this argument. If you want to access it later in the class you can assign the value within `__init__` as in `self._dll_loc = dll_path`

Reply:
@Akshita07 merging my suggestion is not enough to resolve this issue.

Reply:
I have made other required changes as well Jens